### ISSUE TITLE
Report Curly Braces on indices/offsets?

### ISSUE THREAD

**Issue by Targren** [wapmorgan/PhpDeprecationDetector#59]
"Deprecated: Array and string offset access syntax with curly braces is deprecated"

Can a check for this be added (or does it already exist and I'm failing to do something to enable it)?

**Comment by davejenx** [comment]
I agree this would be a good addition. In PHP 8.0, this becomes a fatal error:
`PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported`
So this feature becomes more important now that PHP 7.4 has reached end of life.

**Comment by effeks** [comment]
+1

**Comment by roneymaia** [comment]
+1

**Comment by javyer** [comment]
+1
